android/beagle/src/main/java/br/com/zup/beagle/android/utils/StyleManager.kt

```diff
@@ -46,7 +46,11 @@ class StyleManager(
             view.applyViewBackgroundAndCorner(Color.TRANSPARENT, component)
         } else when (component) {
             is Text -> {
-                applyStyleId(context, component.styleId ?: "", view, component)
+                if (component.styleId == null || component.styleId is Bind.Value) {
+                    applyStyleId(context, (component.styleId?.value ?: "") as String, view, component)
```

Comment:
I'm not sure, but this should work if Moshi is specializing the values.
```suggestion
                if (component.styleId == null || component.styleId is Bind.Value<String>) {
                    applyStyleId(context, (component.styleId?.value ?: ""), view, component)
```